fe/fe-common/pom.xml

```diff
@@ -132,6 +132,15 @@ under the License.
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.aliyun.oss</groupId>
+            <artifactId>aliyun-sdk-oss</artifactId>
+            <version>3.15.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aliyun</artifactId>
+        </dependency>
```

Comment:
If the user does not use Alicloud OSS, can we remove this dependency without causing any exceptions? I'm unsure if there will be any issues.

Reply:
Do you mean to package it together again, but to ask the user to add this package manually?

Reply:
I think it should not affect functions that do not depend on oss